src/Transition/transition_empd.py

```diff
@@ -0,0 +1,177 @@
+# This is a quick and dirty python script to transition the EMPD model.
+# You'll want to use this math in the official transition program.
+# Once you're done that, feel free to delete this.
```

Comment:
@Myoldmopar whenever you get around to porting this to the new Fortran transition program, here's what you'll need to know to transition the inputs for the EMPD model.